xoxzo/cloudpy/cloudpy.py

```diff
@@ -21,17 +21,15 @@ class XoxzoClient:
     Base class to access Xoxzo API.
 
     :param string sid:  Your sid of the xoxzo account.
-        If None, value of the environment variable
-        XOXZO_API_SID will be used.
     :param string auth_token: Your auth_token of the xoxzo account.
-        If None, value of the environment variable XOXZO_API_AUTH_TOKEN
-        will be used.
     '''
 
-    def __init__(self, sid=None, auth_token=None):
+    def __init__(self, sid, auth_token):
         '''
         Initialize and instanceate XoxzoClient object.
         '''
+        self.sid = sid
+        self.auth_token = auth_token
         # you can override api host by setting envrionment
         # variable XOXZO_API_HOST
         api_host = os.environ.get("XOXZO_API_HOST")
```

Comment:
Please remove this one too. You can add a third optional parameter to `__init__()` for `api_host`.